utils/providers.py

```diff
@@ -13,6 +13,10 @@
 import cfme.fixtures.pytest_selenium as sel
 from fixtures.pytest_store import store
 from cfme.web_ui import Quadicon, paginator, toolbar
+from cfme.common.provider import BaseProvider
+from cfme.exceptions import UnknownProviderType
+from cfme.infrastructure.provider import RHEVMProvider, VMwareProvider, SCVMMProvider
```

Comment:
One wonders whether we could do some kind of automatic inspecting of modules and break them out into their own files so that new providers could be added and hence supported without any further effort.